content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -1,390 +1,478 @@
-
-- [Memory Manager KEP: Simulation - how the Memory Manager works? (by examples)][1]
-
-- [Memory Manager KEP: The Concept of Node Map and Memory Maps][2]
-
-- [Memory Manager KEP: How to enable the guaranteed memory allocation over many NUMA nodes?][3]
-
-[1]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1769-memory-manager#simulation---how-the-memory-manager-works-by-examples
-[2]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1769-memory-manager#the-concept-of-node-map-and-memory-maps
-[3]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1769-memory-manager#how-to-enable-the-guaranteed-memory-allocation-over-many-numa-nodes
-[4]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1769-memory-manager#design-overview
-[5]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1769-memory-manager#memory-maps-at-start-up-with-examples
-[6]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1769-memory-manager#memory-maps-at-runtime-with-examples
+---
+title: Utilizing the NUMA-aware Memory Manager
+
+reviewers:
+- klueska
+- derekwaynecarr
+
+content_type: task
+min-kubernetes-server-version: v1.21
+---
+
+<!-- overview -->
+
+{{< feature-state state="beta" for_k8s_version="v1.22" >}}
+
+The Kubernetes *Memory Manager* enables the feature of guaranteed memory (and hugepages)
+allocation for pods in the `Guaranteed` {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}.
+
+The Memory Manager employs hint generation protocol to yield the most suitable NUMA affinity for a pod.
+The Memory Manager feeds the central manager (*Topology Manager*) with these affinity hints.
+Based on both the hints and Topology Manager policy, the pod is rejected or admitted to the node.
+
+Moreover, the Memory Manager ensures that the memory which a pod requests
+is allocated from a minimum number of NUMA nodes.
+
+The Memory Manager is only pertinent to Linux based hosts.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+To align memory resources with other requested resources in a Pod spec:
+
+- the CPU Manager should be enabled and proper CPU Manager policy should be configured on a Node.
+  See [control CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/);
+- the Topology Manager should be enabled and proper Topology Manager policy should be configured on a Node.
+  See [control Topology Management Policies](/docs/tasks/administer-cluster/topology-manager/).
+
+Starting from v1.22, the Memory Manager is enabled by default through `MemoryManager`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+
+Preceding v1.22, the `kubelet` must be started with the following flag:
+
+`--feature-gates=MemoryManager=true`
+
+in order to enable the Memory Manager feature.
+
+## How Memory Manager Operates?
```

Comment:
nit: headings at h2 and higher should be sentence case

As we're updating the page, I'd consider fixing this too.